meta/main.yml

```diff
@@ -1,6 +1,10 @@
 ---
+dependencies:
+  - geerlingguy.repo-epel
+
 galaxy_info:
   author: Luc Stroobant and Dieter Verhelst
```

Comment:
What is proper open-source etiquette on forking & author field?

Leave the current authors seeing as they wrote the role, or would that cause confusion that they may be affiliated with Tag1?

Reply:
I suggest putting their names in the README: `Role originally written by .... now maintained by Tag1 Consulting, Inc.`, and then put your name here in the metadata.